examples/answers-swap.yaml

```diff
@@ -39,5 +39,5 @@ SnapList:
       classic: false
 InstallProgress:
   reboot: yes
-
-
+Drivers:
+  install: no
```

Comment:
It would be nice if we didn't have to do this to every file every time we add a screen wouldn't it... (I don't really have any ideas here). Remember we'll need to do this to the ubuntu-testcases repo too!

Reply:
Indeed, it would be!
One way that I can think of would be to generate the `examples/answers-*.yaml` files based on:
* a single baseline `.yaml` file (where we have "default" values for each item)
* other `.yaml` files that override specific items.
Any thoughts?

I must admit I didn't know about the ubuntu-test-cases / live-server repo's existence :eyes: 
Lesson learned, I guess!